src/backup.ts

```diff
@@ -0,0 +1,369 @@
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const database = spanner.database('my-database');
+   * const backupExpiryDate = new PreciseDate(Date.now() + 1000 * 60 * 60 * 24)
+   * const backup = instance.backup('my-backup', database.formattedName_, backupExpiryDate);
+   * const alreadyExists = await backup.exists();
+   *
+   * // Create new backup only when backup named 'my-backup' does not already exist
+   * if (!alreadyExists) {
+   *   await backup.create();
+   * }
+   */
+  async exists(): Promise<boolean> {
+    try {
+      // Attempt to read metadata to determine whether backup exists
+      await this.getBackupInfo();
+      // Found therefore it exists
+      return true;
+    } catch (err) {
+      if (err.code === status.NOT_FOUND) {
+        return false;
+      }
+      // Some other error occurred, rethrow
+      throw err;
+    }
+  }
+
+  /**
+   * Sets the expiry time of a backup.
+   *
+   * @see {@link #getExpireTime}
+   *
+   * @method Backup#updateExpireTime
+   * @returns {Promise<Backup>} when resolved, the backup's expire time will
+   *     have been updated.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const myBackup = instance.backup('my-backup');
+   * const newExpireTime = new PreciseDate(Date.now() + 1000 * 60 * 60 * 24);
+   * await myBackup.updateExpireTime(newExpireTime);
+   */
+  updateExpireTime(expireTime: PreciseDate): Promise<Backup>;
+  updateExpireTime(
+    expireTime: PreciseDate,
+    callback: UpdateExpireTimeCallback
+  ): void;
+  updateExpireTime(
+    expireTime: PreciseDate,
+    callback?: UpdateExpireTimeCallback
+  ): void | Promise<Backup> {
+    this.expireTime = expireTime;
+
+    const reqOpts: databaseAdmin.spanner.admin.database.v1.IUpdateBackupRequest = {
+      backup: {
+        name: this.formattedName_,
+        expireTime: expireTime.toStruct(),
+      },
+      updateMask: {
+        paths: ['expire_time'],
+      },
+    };
+    return this.request<databaseAdmin.spanner.admin.database.v1.IBackup>(
+      {
+        client: 'DatabaseAdminClient',
+        method: 'updateBackup',
+        reqOpts,
+      },
+      err => {
+        callback!(err, err ? undefined : this);
+      }
+    );
+  }
+
+  /**
+   * Deletes a backup.
+   *
+   * @method Backup#deleteBackup
+   * @returns {Promise<void>} when resolved, the backup will have been deleted.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const myBackup = instance.backup('my-backup');
+   * await myBackup.deleteBackup();
+   */
+  deleteBackup(): Promise<void>;
+  deleteBackup(callback: DeleteBackupCallback): void;
+  deleteBackup(callback?: DeleteBackupCallback): void | Promise<void> {
```

Comment:
Just `delete()`, so that it's `backup.delete()` as opposed to `backup.deleteBackup()`.